### PR TITLE
Add documentation for how to administer Read the Docs for documentation and update Plone Sphinx Theme configuration

### LINKED ISSUE #10: Document how to set up RTD pull request previews
https://github.com/readthedocs/actions/tree/v1/preview

### LINKED ISSUE #1604: Unpin spinxcontrib-*help requirements after theme upgrade
In `requirements.txt` remove the temporary pins after we upgrade to the latest theme.

```
sphinxcontrib-applehelp==1.0.4  # Documentation only, temporary pin
sphinxcontrib-devhelp==1.0.2  # Documentation only, temporary pin
sphinxcontrib-htmlhelp==2.0.1  # Documentation only, temporary pin
sphinxcontrib-qthelp==1.0.3  # Documentation only, temporary pin
sphinxcontrib-serializinghtml==1.1.5  # Documentation only, temporary pin
```

### PR DESCRIPTION
**Pull request by stevepiercy**
Closes https://github.com/plone/plone-sphinx-theme/issues/10

Closes #1604

Depends on merge of https://github.com/plone/volto/pull/6030 because its `custom.css` file pollutes default styles, turning the `plone.api` docs into an unreadable mess, as shown in https://plone6--1667.org.readthedocs.build/plone.api/api/user.html.

### CODE REVIEW

File: docs/contributing/documentation/admins.md [hunk 1]
```diff
@@ -84,49 +84,42 @@ To make it easier for other contributors to work with your project, update the f
...
+1.  Click {guilabel}`Continue`.
+1.  In the {guilabel}`Add project` screen, you can configure basic project settings, including its {guilabel}`Name`, {guilabel}`Repository URL`, {guilabel}`Default branch`, and {guilabel}`Language`.
+    The defaults are usually accurate.
+1.  Click {guilabel}`Next`.
+1.  A sample `.readthedocs.yaml` file is suggested, if you have not already added one.
+1.  Click {guilabel}`Finish`.
+    Read the Docs will redirect you to the project details, and start building the docs.
+
+For most Plone projects, you will not want to Read the Docs to publish the `latest` or other specific versions.
+Plone projects currently self-host their official documentation.
```
Comment (jensens) [inline docs/contributing/documentation/admins.md]: Sorry, I do not get exactly what this two lines are supposed to tell me.

Comment (stevepiercy) [reply docs/contributing/documentation/admins.md]: We already have a public version that gets indexed by search engines. We don't want that for pull request previews. I also realized this section was kind of incomplete in regards to activating and hiding versions. Here's a revision.

````suggestion
Many Plone projects currently self-host their official documentation at {doc}`/index`.
These projects get indexed by search engines.

For pull request previews, unsupported branches or versions, or other situations, you most likely do not want search engines to index your documentation.
In this case, you can create a custom {file}`robots.txt` file to discourage, but not absolutely prevent, search engine indexing.
You can configure Sphinx to copy the {file}`robots.txt` file.
However, if you want to have two versions of a {file}`robots.txt` file—say one that allows indexing of your official documentation and another that discourages indexing—you can configuration your automation to copy it into place.

```shell
cp some-path/block-robots.txt docs-root-path/robots.txt
```

You can also configure which branches to activate or hide through Read the Docs.

```{seealso}
-   [`robots.txt` support](https://docs.readthedocs.io/en/stable/reference/robots.html)
-   [Automation rules](https://docs.readthedocs.io/en/stable/automation-rules.html)
-   [Versions](https://docs.readthedocs.io/en/stable/versions.html)
-   [Managing versions automatically](https://docs.readthedocs.io/en/stable/guides/automation-rules.html)
```
````

Comment (stevepiercy) [reply docs/contributing/documentation/admins.md]: @jensens I revised this section and improved the flow. Please let me know.

**Review by davisagli — DISMISSED** [review]

**Review by jensens — DISMISSED** [review]

**Comment by stevepiercy** [comment]
I broke out pull request preview stuff because we need it now, since we purged netlify.

**Pull request merged**